docker/Dockerfile.ci_hexagon

```diff
@@ -16,20 +16,17 @@
 # under the License.
 
 # CI docker Hexagon env
-# tag: v0.01
-FROM tvmcihexagon/ci-hexagon-base:v0.01_SDK4.2.0.2
+# tag: v0.02
+FROM tvmcihexagon/ci-hexagon-base:v0.02_SDK4.2.0.2
```

Comment:
Does the name "SDK4.2.0.2" mean anything?  Is the plan to use SDK 4.5.0.3 in this image?